config/locales/devise/en.yml

```diff
@@ -131,8 +131,8 @@ en:
         link: Use a personal key instead
         text_html: Don’t have access to your phone? %{link}.
       personal_key_header_text: Enter your personal key
-      personal_key_prompt: You can use this personal key once. If you still need a
-        code after signing in, go to your account settings page to get a new one.
+      personal_key_prompt: You can use this personal key once. Once you enter it,
```

Comment:
How about "After you enter it..."? (double-once ;)

Reply:
I would say something like  'Each time you use it, ...' since the user 'enters' it when confirming it and this is not a one time (or two time) thing.

Reply:
Generally, we should design our content before we make code changes...good catch, @mryenq 

Not going to put a ton into this since personal key will be changing more significantly. 

>~Once you enter it~ 

> After you enter it